src/components/LayoutWithRouting.tsx

```diff
@@ -24,21 +24,28 @@ const Route: React.FC<RouteProps> = ({ Component, ...pathProps }) => {
   )
 }
 
-const LayoutWithRouting: React.FC = () => (
-  <>
-    <GlobalStyle />
-    <Navbar default />
-    <MainContainer>
-      <Router primary={false}>
-        <Route default Component={HomeView} />
-        <Route path={routes.video()} Component={VideoView} />
-        <Route path={routes.search()} Component={SearchView} />
-        <Route Component={BrowseView} path={routes.browse()} />
-        <Route Component={ChannelView} path={routes.channel()} />
-      </Router>
-    </MainContainer>
-  </>
-)
+const LayoutWithRouting: React.FC = () => {
+  globalHistory.listen(({ action }) => {
```

Comment:
This need to go into `useEffect`

Reply:
@Gamaranto this doesn't work. You're currently calling unsubscribe function right after subscribing